tests/api/test_dtls.c

```diff
@@ -595,3 +595,54 @@ int test_wolfSSL_dtls_cid_parse(void)
 #endif
     return EXPECT_RESULT();
 }
+
+int test_dtls13_epochs(void) {
```

Comment:
Nice to Have: a test with a complete handshake and/or a test with DTLS fuzzer.

Reply:
Feasible once we have a reliable framework for generating test messages directly. We can consider this for next major TLS improvements.